client/state/terms/reducer.js

```diff
@@ -9,15 +9,80 @@ import keyBy from 'lodash/keyBy';
  * Internal dependencies
  */
 import {
+	DESERIALIZE,
 	TERMS_RECEIVE,
-	DESERIALIZE
+	TERMS_REQUEST,
+	TERMS_REQUEST_FAILURE,
+	TERMS_REQUEST_SUCCESS,
+	SERIALIZE
 } from 'state/action-types';
 import { isValidStateWithSchema } from 'state/utils';
+
+import {
+	getSerializedTaxonomyTermsQuery
```

Comment:
We could probably get away with shortening this to `getSerializedTermsQuery`, would you agree?